tests/e2e/sample-applications/parks-app/manifest.yaml

```diff
@@ -46,7 +46,7 @@ items:
           from:
             kind: ImageStreamTag
             namespace: openshift
-            name: nodejs:10
+            name: nodejs:10-ubi8
```

Comment:
This will break anyone developing on OCP <4.7. We need to potentially version this file for different OCP versions